### ISSUE TITLE
RuntimeException when installing

### ISSUE THREAD

**Issue by magikid** [snipe/snipe-it#4523]
#### Expected Behavior (or desired behavior if a feature request)

Following the [installation documentation](https://snipe-it.readme.io/docs/installation) from start to finish throws no errors.

-----

#### Actual Behavior

I am following the [documentation](https://snipe-it.readme.io/) to install Snipe-IT on my own server.  In the configuration step, I was instructed to copy the `.env.example` file to `.env`.  When I got to the [Generate App Key step](https://snipe-it.readme.io/docs/generate-your-app-key), I ran `php artisan key:generate` and a Runtime Exception was thrown:

>   [RuntimeException]                                                                        
>  The only supported ciphers are AES-128-CBC and AES-256-CBC with the correct key lengths.  

Every time I tried to run `php artisan ...` the exception would be thrown.  After researching it, the issue is due to `.env.example` having a default `APP_KEY` value of `ChangeMe`.  That is an invalid key and so `artisan` is throwing the exception.  Changing it to a valid key allowed me to generate my own key.

-----

#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? **Fresh install**
- Version of Snipe-IT you're running **fcc87b3219ca5ecc82a75602ac61ad0278572b6c**
- Version of PHP you're running **7.2**
- Version of MySQL/MariaDB you're running **5.6**
- What OS and web server you're running Snipe-IT on **freebsd 10.3, apache24**
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) **manual installation**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error **N/A**
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error **See above**
- If a stacktrace is provided in the error, include that too. **See above**
- Any errors that appear in your browser's error console. **See above** 
- Confirm whether the error is reproduceable on the demo: https://snipeitapp.com/demo. **N/A**
- Include any additional information you can find in `app/storage/logs` and your webserver's logs. **N/A**
- Include what you've done so far in the installation, and if you got any error messages along the way.  **See above**
- Indicate whether or not you've manually edited any data directly in the database  **No**

**Comment by magikid** [comment]
Looking at the other `.env` files in the repo, I see that some of them include a default key that would work with `artisan` and some don't.  I'd be happy to standardize that and put in a pull request for it.

**Comment by snipe** [comment]
> Every time I tried to run php artisan ... the exception would be thrown. After researching it, the issue is due to .env.example having a default APP_KEY value of ChangeMe. That is an invalid key and so artisan is throwing the exception. Changing it to a valid key allowed me to generate my own key.

That's what `php artisan key:generate` does. It generates a key. We leave it as `ChangeMe` specifically so that it *does* error out if you're not set a custom key, since that key is used in encryption and should be unique to every install.

It sounds like you've resolved this, but I'd normally recommend running the standard commands that work when running into these types of things:

```
php composer dump
php composer install --no-dev --prefer-source
php artisan config:cache
```

**Comment by magikid** [comment]
I understand the importance of having a unique for each install.  My issue is that without having a valid `APP_KEY` in the `.env` file to begin with, I couldn't run the command to generate a new `APP_KEY`.  All I knew was that ChangeMe was an invalid key but there was no further documentation showing me an example of what a valid `APP_KEY` should be.
 
I guess my issue is that there is no mention that this is a known problem in the documentation.  I don't regularly use this stack so how was I, as a new user, to know that I needed to update this exact setting to get any further code to work?

What if I updated the documentation to mention that this is a known issue and a solution?  Or added a commented out APP_KEY so that other users knew what format to put the APP_KEY in?  Would that be an acceptable solution for the project?
